src/main/java/de/rytrox/bedwars/utils/Countdown.java

```diff
@@ -0,0 +1,51 @@
+package de.rytrox.bedwars.utils;
+
+import de.rytrox.bedwars.Bedwars;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerJoinEvent;
+import org.bukkit.plugin.java.JavaPlugin;
+import org.bukkit.scheduler.BukkitTask;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Countdown implements Listener {
+
+    private BukkitTask task;
+    private int minPlayerSize = 1;
+
+    @EventHandler
+    public void onPlayerJoinListener(PlayerJoinEvent event) throws InterruptedException {
+        int playerOnlineSize = Bukkit.getOnlinePlayers().size();
+        if(playerOnlineSize > minPlayerSize) {
+            this.start();
+        }
+    }
+
+    private void start() {
+        AtomicInteger timer = new AtomicInteger(10);
+        task = Bukkit.getServer().getScheduler().runTaskTimer(JavaPlugin.getPlugin(Bedwars.class), () -> {
+
+                for (Player player : Bukkit.getOnlinePlayers()) {
+                    if(timer.get() > 0) {
+                        player.sendTitle("" + timer.get(), "SECONDS LEFT");
+                    } else {
+                        player.sendTitle("START","BEDWARS");
+                        task.cancel();
+                    }
+                }
+                timer.set(timer.get() - 1);
```

Comment:
```suggestion
                int currentTime = timer.decrementAndGet();
                if(currentTime > 0) {
                    Bukkit.getOnlinePlayers().forEach((player) -> player.sendTitle(String.valueOf(currentTime), "SECONDS LEFT"));
                } else {
                    Bukkit.getOnlinePlayers().forEach((player) -> player.sendTitle("START", "BEDWARS"));
                    task.cancel();
                }
```
AtomicInteger hat sogar eine exklusive Methode, die die Zahl um 1 runterzählt und dann die aktuelle Zahl als int zurückgibt. Das wäre hier doch total praktisch!

Den Task solltest du nicht in der ForEach canceln...
Der würde sonst die Cancel Methode für jeden Spieler, der online ist, einmal aufrufen. Das wäre zu viel